src/components/ImageGallery/ImageGallery.tsx

```diff
@@ -1223,4 +1226,32 @@ export const ImageGallery = <
   );
 };
 
+/**
+ * Clamping worklet to clamp the scaling
```

Comment:
Hate this re-ordering for styleguidist, takes away from the use in order principals